bridges/otellogr/logsink.go

```diff
@@ -10,13 +10,27 @@
 // way:
 //
 //   - Message is set as the Body using a [log.StringValue].
-//   - TODO: Level
+//   - Level is transformed and set as the Severity. The SeverityText is not
+//     set.
 //   - KeyAndValues are transformed and set as Attributes.
 //   - The [context.Context] value in KeyAndValues is propagated to OpenTelemetry
 //     log record. All non-nested [context.Context] values are ignored and not
 //     added as attributes. If there are multiple [context.Context] the last one
 //     is used.
 //
+// The V-level is transformed by using the [WithLevelSeverity] option. If option is
+// not provided then V-level is transformed in the following way:
+//
+//   - logr.Info and logr.V(0) are transformed to [log.SeverityInfo].
+//   - logr.V(1) is transformed to [log.SeverityDebug].
+//   - logr.V(2) and higher are transformed to [log.SeverityTrace].
+//
+// If possible, look at alternative log bridges that provide less abstract log
```

Comment:
This comment is a bit weird. Is that intended at folks using the library, and telling them to look at how other bridges are doing it, or is that a todo?

Reply:
Blame is on me.

I will add a context why I proposed it. Imagine you have an application that uses `logr` for emitting logs that is configured to have `zap` as its backend. It might be better if this application bridges the `zap` to use `otelzap` than `otellogr` to get more precise translations based on their current usage...

However, after thinking the second time I think it may indeed be a strange comment and probably it is better to remove it as it would probably just cause more confusion.

TL;DR;
Lets remove it.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6206/commits/b171aad6ab794e21a60f27c355ea87fe38e7175d